hack/e2e.sh

```diff
@@ -33,11 +33,30 @@ oc secrets link default ci-pull-secret --for=pull
 oc new-app -f template/cincinnati.yaml \
   -p IMAGE="${IMAGE_FORMAT%/*}/stable" \
   -p IMAGE_TAG=deploy \
-  -p GB_CINCINNATI_REPO="redhat/openshift-cincinnati-test-public-manual" \
   -p GB_CPU_REQUEST=50m \
   -p PE_CPU_REQUEST=50m \
   -p RUST_BACKTRACE="1" \
-  -p GB_REGISTRY_CREDENTIALS_PATH="" \
+  -p GB_PLUGIN_SETTINGS='
```

Comment:
nitpick: params are growing large, lets put these in a file and use `oc new-app --param-file`?

Reply:
It's a bit hard to justify, given that we're talking about code in *hack*. We could move it to dist and take it more serious though.

Reply:
Do you consider this a blocker?

Reply:
No, lets rework this later